_posts/blog/2019-11-04-checkstyle-import-control.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "Checking API use"
+---
+
+Accumulo follows [SemVer] across versions with the declaration of a public API.  Code not in the public API should be
+considered unstable, at risk of changing between versions.  The packages included in the public API are [listed on the website][api]
+but may not always be considered when developing using Accumulo code.  This blog post explains how to setup a Maven project
+to automatically detect when Accumulo 2.0 code used in the project is outside of the public API.
+
```

Comment:
I think it would be really useful to add a sentence like the following.  This info is very important.

```
The techniques described in this blog post only work for Accumulo 2.0 and later.  Do not use these with 1.X versions because those version have `impl` packages inside API packages that must be excluded.  The needed exclusions for 1.X are not supported by import control.
```

Reply:
You are right, but I don't think we need to try to explain why.  I added a shortened version of this warning.